app/javascript/components/upload/MetadataStep.jsx

```diff
@@ -39,6 +40,7 @@ function MetadataForm({
   const userState = useContext(UserContext)
   const featureFlagState = serverState.feature_flags
   const conventionRequired = featureFlagState && featureFlagState.convention_required
+  const explainerImg = isAnnDataExperience ? annDataMetadataExplainerImage : metadataExplainerImage
```

Comment:
Great for me to see how this is done (I would have done something much less elegant/DRY).